src/assets/translations/fr/main.json

```diff
@@ -576,6 +624,11 @@
       "withdrawFee": "Il y aura des frais de carburant distincts pour effectuer un retirait.",
       "disclaimer": "Les demandes ultérieures seront combinées et réinitialiseront le temps de retrait.",
       "withdrawalFee": "Frais de retrait",
+      "status": {
+        "pending": "Votre transaction a été diffusée et est en attente de confirmation.",
+        "success": "Vous avez effectué un retrait avec succès de %{opportunité}.",
```

Comment:
Interpolation variables shouldn't be translated

Reply:
Addressed in https://github.com/shapeshift/web/pull/2220/commits/b21538aa8648db3efa253c57ceacc30df3f9871b